net-nntp/slrn/slrn-1.0.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P="${P}a"
+
+DESCRIPTION="A s-lang based newsreader"
+HOMEPAGE="http://slrn.sourceforge.net/"
+SRC_URI="http://jedsoft.org/releases/${PN}/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="canlock nls ssl uudeview"
+
+RDEPEND="virtual/mta
```

Comment:
Please sort the unconditional dependencies lexically, it's easier to review when we can expect the order.